storage/backup/backup-service/src/handlers.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+use anyhow::Result;
+use libra_crypto::hash::HashValue;
+use libra_logger::prelude::*;
+use libra_types::transaction::Version;
+use libradb::backup::BackupHandler;
+use std::convert::Infallible;
+use warp::{filters::BoxedFilter, reply::Reply, Filter, Rejection};
+
+fn get_state_range_proof(
+    backup_handler: &BackupHandler,
+    version: Version,
+    end_key: HashValue,
+) -> Result<Box<dyn Reply>> {
+    let bytes = lcs::to_bytes(&backup_handler.get_account_state_range_proof(end_key, version)?)?;
+    Ok(Box::new(bytes))
+}
+
+/// Return 500 on any application error.
```

Comment:
not sure about the standard term here. maybe "application" -> "server"?

Reply:
it's probably Facebook thrift terminology :P will update.